[nnc] Updated generic error message with info about turning off the fuser

navahgar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64329
* #64328
* #64319
* #64318
* #64317
* __->__ #64316

Differential Revision: [D30683942](https://our.internmc.facebook.com/intern/diff/D30683942)

navahgar committed:
[nnc] Updated generic error message with info about turning off the fuser

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30683942).

bertmaher commented on torch/csrc/jit/tensorexpr/kernel.cpp:
```diff
@@ -67,10 +67,16 @@ namespace jit {
-  return s + " " + generic_error_message;
+  static const std::string generic_error_message =
+      "This error occured in the fuser. You can turn off the fuser by setting "
+      "torch._C._jit_override_can_fuse_on_cpu to false";
```
Maybe "You can turn off the fuser with torch._C._jit_override_can_fuse_on_cpu(False)"
Although it's kind of annoying if it's the GPU fuser but 🤷

navahgar replied:
Updated that. 

Good point regarding GPU fuser. Should we provide one flag to turn both off?

navahgar committed:
Update on "[nnc] Updated generic error message with info about turning off the fuser"


Differential Revision: [D30683942](https://our.internmc.facebook.com/intern/diff/D30683942)

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30683942).
